data/lang/jp.json

```diff
@@ -30,28 +30,28 @@
-  "first_join" : "Hello! Bot noticed that you or someone else invited the bot to your server called **_SSS_**. If proper permissions were given to the bot, bot might have created a role called `PackPackMod`. This is because bot limits user to use its command by roles. If you want to adjust assigned role data of the bot, assign `PackPackMod` role to yourself, and call `p!idset` command",
-  "error_api" : "Uh oh, it seems that Discord team has problem in there server. It can be temporary outage, so please wait for few minutes and try again later\n\nCheck Discord status in this link : https://discordstatus.com/",
-  "require_server" : "Sorry! This command must be used in server, not here!",
-  "bot_issue" : "There was technical issue, or unknown problem that made bot go offline. %s is now online",
-  "bot_online" : "%s is now online",
-  "bot_offline" : "%s will be offline %s",
-  "bot_bug" : "temporarily due to bug fix",
-  "bot_update" : "temporarily due to updates",
-  "bot_api" : "temporarily due to API updates",
-  "bot_maintenance" : "temporarily due to maintenance",
-  "bot_end" : "permanently due to out of service. Thanks for using this bot",
-  "search_prev10" : "Previous 10 Pages",
-  "search_prev" : "Previous Page",
-  "search_next" : "Next Page",
-  "search_next10" : "Next 10 Pages",
-  "search_list" : "Select list to pick data",
-  "data_customtrea" : "This embed applies customized treasure data",
-  "data_customtalent" : "This embed applies talents",
+  "needroleperm" : "ボットはロールを作成する権限がないことに気が付きました。ボットを動作させるには、ボットにロールの作成を許可する必要があります。そうしないとボットはどのユーザーにも反応しません。\n\nボットがモデレーターを認識できるようには`PackPackMod`という初期ロールを追加する必要があります。通常、モデレーターは自分自身にロールを割り当てる権限を持っているため、このロールを割り当てる必要があります。その後`p!idset`と入力して登録されているロールデータを変更します。例えば、モデレーターのロールを`PackPackMod`からサーバーのモデレーターロールに変更したり、他のメンバーがコマンドを使えるように設定できます。\n\nこのメッセージは**_**というサーバーからボットによって自動で送信されました。",
+  "maxrole" : "ボットはこのサーバーがすでに最大ロール数(250)に達しているため、必要な新しいロールを追加できず、ボットが正常に機能できませんでした。ロールを１つ削除して、このボットを再度招待して下さい。\n\nこのメッセージは**_**というサーバーからボットによって自動で送信されました。",
+  "no_perm" : "次のサーバーからのメッセージです : **_SSS_**\n\nボットはコマンドを使用したチャンネルでメッセージを送信できないことに気が付きました。　このサーバーの所有者またはモデレーターに連絡してボットの権限を調節するか、あなたがその一人である場合は、このボットの権限を調節してメッセージを送信できるようにしてください。",
+  "no_permch" : "次のサーバーからのメッセージです : **_SSS_**\n\nボットは**#_CCC_**というチャンネルでメッセージを送信できないことに気が付きました。このサーバーの所有者またはモデレーターに連絡してボットの権限を調節するか、あなたがその一人である場合は、このボットの権限を調節してメッセージを送信できるようにしてください",
+  "first_join" : "こんにちは!ボットはあなたか他の人が**_SSS_**というサーバーに招待したことを認識しました。ボットに適切な権限が与えられていれば`PackPackMod`というロールを作成したはずです. これは、ボットがロールによってユーザーにコマンドの使用を制限しているためです。ボットに割り当てられたロールを調整する場合は`PackPackMod`ロールを自分に割り当て`p!idset`コマンドを呼び出します。",
+  "error_api" : "おっと、Discordチームのサーバーに問題が発生しているようです。一時的に停止している可能性があるので、数分待ってから再度お試しください。\n\nこのリンクでDiscordのステータスを確認してください : https://discordstatus.com/",
+  "require_server" : "ごめんなさい!このコマンドは、ここではなくサーバーで使用する必要があります!",
+  "bot_issue" : "技術的な問題または未知の問題によりボットがオフラインになりました。 %s は現在オンラインです",
+  "bot_online" : "%s は現在オンラインです",
+  "bot_offline" : "%s はオフラインになります %s",
```

Comment:
This is bit complicated. First `%s` is mention of the bot, and second `%s` is the reason. I have prepared several reasons to make bot offline; Bug fix, Update, API update, Maintenance. Below texts, `bot_bug`, `bot_update`, `bot_api`, `bot_maintenance`, and `bot_end` are used for stating reason. Considering how you translated the texts below, it would be normal to see

```
@PackPack は[バグ修正のため一時的に]オフラインになります
@PackPack は[アップデートのため一時的に]オフラインになります
@PackPack は[API更新のため一時的に]オフラインになります
@PackPack は[メンテナンスのため一時的に]オフラインになります
```

So my suggestion could be `%s は%sオフラインになります`. However, we still have one problem. We can't handle `bot_end` as we can't really apply text into my suggested text. You may need to figure out other way. Buuut, I think we can worry about this later because it will be never used anyway